docker-images/Makefile

```diff
@@ -1,6 +1,6 @@
 SUBDIRS=java-base kafka-base zookeeper kafka kafka-connect kafka-connect/s2i \
         stunnel-base zookeeper-stunnel kafka-stunnel entity-operator-stunnel \
-        test-client
+        test-client kafka-mirror-maker
```

Comment:
Would it be easier to call it just mirror-maker? We have quite a lot of Kafka's already and it doesn't really add the value here. Just an idea.